requirements.apt

```diff
@@ -0,0 +1 @@
+libpq-dev
```

Comment:
I would have added a comment here to explain this file but comments are not allowed in here. Suffice to say that Tsuru uses this file to install any packages that the app requires, one on each line.